Common/cpp/ReanimatedRuntime/WorkletRuntime.h

```diff
@@ -18,6 +18,8 @@ using namespace react;
 
 namespace reanimated {
 
+enum WorkletRuntimeType { WithLocking, WithoutLocking };
```

Comment:
Why do we use enum instead of boolean `supportsLocking_`?

Reply:
I don't know why kmagiera did it like that, but I can agree that simple boolean is better 👍